Cargo.toml

```diff
@@ -11,28 +11,21 @@ keywords = ["routing", "bgp"]
 license = "BSD-3-Clause"
 
 [dependencies]
-
-# Optional dependencies which, if included, introduce additional functionality.
-arbitrary = { version = "1", optional = true, features = ["derive"] }
-bcder = { version = "0.7.0", optional = true }
-serde = { version = "1.0.95", optional = true, features = ["derive"] }
-
-
-bytes = { version = "1.1.0", optional = true }
-chrono = { version = "0.4", optional = true }
-const-str = { version = "0.5.3", optional = true, features = ["case"] }
-log = { version = "0.4.17", optional = true }
-
-octseq = { version = "0.3.1", features = ["bytes"], optional = true }
-tokio = { version = "1", features = ["sync", "rt"], optional = true }
+arbitrary   = { version = "1", optional = true, features = ["derive"] }
+bcder       = { version = "0.7", optional = true }
+bytes       = { version = "1", optional = true }
+chrono      = { version = "0.4.20", optional = true }
+const-str   = { version = "0.5", optional = true, features = ["case"] }
+log         = { version = "0.4.4", optional = true }
+octseq      = { version = "0.3.1", features = ["bytes"], optional = true }
```

Comment:
Maybe swap option and features here (and also for tokio), so that optional is always first which, I think, is the more important information here.

Reply:
Yes. Did that for the other PR which obsoletes this one.